### DISCUSSION TITLE
Disable toc with mkdocs-material v7.1.4

### DISCUSSION THREAD

**Discussion by evoulitas** [squidfunk/mkdocs-material#2644] [Q&A] [answered]
Hi,
I am trying to completely remove the toc for all pages and what used to work for me with mkdocs-material==6.1.2 [1] does not seem to work now with version 7.1.4 :/
I have also tried disabling it under the markdown_extensions [2] but this does not work either.
Thank you very much for the support

[1]
.md-sidebar--secondary {
  display: none;
}

[2] 
markdown_extensions:
  - toc:
      toc_depth: 0

**Comment by squidfunk** [comment, marked as answer]
I've just verified [2]:

``` yaml
markdown_extensions:
  - toc:
      toc_depth: 0
```

... and it works, i.e. the table of contents is removed on all pages:

![screenshot-localhost-8000-getting-started-1620556568241](https://user-images.githubusercontent.com/932156/117568899-34731280-b0c3-11eb-84a3-17d018108374.png)

**Comment by evoulitas** [comment]
Oups..you re right! I was editing the wrong yml:/
Thank you so much for the prompt reply =)
